frontend/src/store/modules/shoots/getters.js

```diff
@@ -131,6 +138,16 @@ export function getSortVal (rootGetters, item, sortBy) {
 
 export default {
   filteredItems (state) {
+    if (state.freezeSorting) {
+      // When state is freezed, do not include new items
+      return compact(map(state.uidsAtFreeze, freezedUID => {
+        const storeItem = find(state.filteredShoots, ['metadata.uid', freezedUID])
+        if (storeItem) {
+          return storeItem
+        }
+        return state.freezedStaleShoots[freezedUID]
```

Comment:
in my local test I had a case where no item was found in `state.freezedStaleShoots` for `freezedUID`